helmfile.d/values/rclone/sync.yaml.gotmpl

```
@@ -46,12 +49,15 @@ targets:
     destinationCrypt: {{ $crypt | get "enabled" false }}
     {{- $destinations = get "destinationType" $sync.destinationType . | append $destinations }}
     destinationType: {{ get "destinationType" $sync.destinationType . }}
+    destinationPath: {{ get "destinationPath" "" . }}
```

Spoke with @aarnq IRL, and we agreed on using the `|` "format" for `.` as well

```suggestion
    destinationPath: {{ . | get "destinationPath" "" }}
```

---

I have now refactored the usage of get in the rclone values files. But nowhere else, the other files felt out of scope.